platform/ext/target/nordic_nrf/nrf5340/partition/flash_layout.h

```diff
@@ -67,7 +67,7 @@
 /* Flash layout info for BL2 bootloader */
 #define FLASH_BASE_ADDRESS                  (0x00000000)
 
-#ifdef BL2
+
```

Comment:
Nit: This line isn't added for the 91 file.

Reply:
fixed thanks